packages/components/data-table/src/data-table.story.js

```diff
@@ -23,7 +23,7 @@ import NumberInput from '../../inputs/number-input';
 const items = [
   {
     id: '5e188c29791747d9c54250e2',
-    name: 'Morgan Bean',
+    name: 'Barry Allen - aka Flash - aka Fastest Man Alive',
```

Comment:
😆 

(I have no objections)

Reply:
😆 Just wanted to show the UX/UI people how would a long name be affected by the truncation feature